modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/apirequest/Output.scala

```diff
@@ -117,6 +116,5 @@ final case class JsonOutput(jsonPath: String) extends ApiOutput[Json] {
       case other => other.leftMap(JsonPathException.apply)
     }
 
-  def parseResponse(response: String): Either[Throwable, Json] =
-    parse(response)
+  def parseResponse(response: String): Either[Throwable, Json] = io.circe.jackson.enrich.parse(response)
```

Comment:
It feels that all we need here is
```suggestion
  private[this] final val jsonFactory: JsonFactory = new JsonFactory(CirceUtils.mapper)
  def parseResponse(response: String): Either[Throwable, Json] = {
    val parser = jsonFactory.createParser(input)
    CirceUtils.mapper.readValue(parser(input), classOf[Json])
  }
```

And we don't need the rest of the magic with creating own parsers.